client/src/pages/draft-board-ui.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState, memo } from 'react';
+import {DragDropContext, Droppable} from 'react-beautiful-dnd';
+import { makeStyles } from '@material-ui/core/styles';
+import {Grid} from '@material-ui/core';
```

Comment:
I can't help but notice some of the differences in formatting (e.g. curly braces and spacing) with the rest of the client repository. How did you trick my pre-commit hooks?! :^)

Reply:
Mannn, I assumed the linting config would kick in for the files upon saving. I guess not but this will be a quick fix! Thanks @ugur-akin .

Reply:
I thought so too! Either way, it's not a real concern and no need to prioritize. I was just surprised to see this.